views/request_password_reset.ejs

```diff
@@ -78,12 +78,12 @@
       </header>
       <div class="section-body">
         <h1 class="email-heading section-body">Password Change Request!</h1>
-        <p class="email-message">Hello <%= first_name %>,</p>
+        <p class="email-message">Hello <%= user.first_name %>,</p>
         <p class="email-message">
           We noticed you requested to change your Aphia password. If it's not you, please ignore.
         </p>
         <p>Else, click here to continue.
-            <a href="https://aphia.onrender.com/password-reset/user_id_any_identification_form">Click here</a>
+            <a href="https://blackholemain.onrender.com/api/reset-password/?email=<%=user.email%>&token=<%=token%>">Click here</a>
```

Comment:
Remember the click here link should point to the front end not the backend URL. like react_frontend_url/reset-password/user_email...
update it
You can set the localhost of their react app for now

Reply:
we need to test it from the backend first